var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -309,10 +320,9 @@ def make_defs(self):
         # When mixing compilers make sure that
         # $SPACK_ROOT/lib/spack/env/<compiler> have symlinks with reasonable
         # names and hack them inside lib/spack/spack/compilers/<compiler>.py
-        make_defs = [
-            "CC={0}".format(spack_cc),
-            "FC={0}".format(spack_fc),
-        ]
+        make_defs = ["CC={0}".format(spack_cc)]
+        if "+fortran" in self.spec:
```

Comment:
```suggestion
        if "~fortran" not in self.spec:
```

We should set `FC` for OpenBLAS < 0.3.21